bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYBJets_HT-incl_Zpt-100to200/DYBJets_HT-incl_Zpt-100to200_run_card.dat

```diff
@@ -0,0 +1,271 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************
+# Running parameters
+#*******************
+#
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false.     = gridpack  !True = setting up the grid pack
```

Comment:
Isn't `gridpack` supposed to be `true`?

Reply:
Hello @SanghyunKo! As I remember it is set to true automatically while using gridpack production script.